docs/user/rest-api.rst

```diff
@@ -859,6 +859,28 @@ Delete Floor Plan
 
     DELETE /api/v1/controller/floorplan/{pk}/
 
+List Indoor Coordinates of a Location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    this endpoint returns device coordinates from the lowest positive
+    floor by default. If no positive floor exists, it returns coordinates
+    from the highest negative floor instead.
```

Comment:
```suggestion
    This endpoint returns device coordinates from the first floor above
    ground (lowest non-negative floors) by default. If a location only has 
    negative floors (e.g. underground parking lot), then it will return the 
    closest floor to the ground (maximum negative floor). 
```